### ISSUE TITLE
ReferenceError: can't find variable require

### ISSUE THREAD

**Issue by vinipachecov** [storybookjs/require-context.macro#31]
Hi there!

I was following the storybook tutorial but using it with react native. Is it working for RN?
1. I installed require-context.macro
2. I installed babel-plugin-macros
3. Configured storybook to load stories

Here is my config file for storybook
 
![image](https://user-images.githubusercontent.com/2838834/67759775-d7eb5d80-fa1e-11e9-9c2f-25815dadc635.png)

![image](https://user-images.githubusercontent.com/2838834/67759826-ee91b480-fa1e-11e9-963a-8c14fc254f7a.png)

My package.json

![image](https://user-images.githubusercontent.com/2838834/67759889-0f5a0a00-fa1f-11e9-909c-79d3409f69d7.png)

![image](https://user-images.githubusercontent.com/2838834/67760172-98714100-fa1f-11e9-8ff1-ba59c059546f.png)

Steps to reproduce:
- Create react native app with react-native init
- Install and configure storybook
- install require-context.macro
- configure stories as in storybook tutorial
- see error

**Comment by kylemh** [comment]
Are your stories using CSF or old API?

**Comment by vinipachecov** [comment]
I’m using the old API, using storiesOf() because this error was occuring only when running react-native run-ios.
Storybook is running fine.

**Comment by kylemh** [comment]
Why would run Storybook inside the native app 🤔 

Do your storyshots work, or no?

**Comment by vinipachecov** [comment]
Well, on React Native it renders the components in the app.
So to run storybook you have to run inside the app.
At least this are the instructions in the official docs.
Storyshots isn’t only for testing?

**Comment by kylemh** [comment]
Good to know about RN, Storyshots is definitely only for testing though.

You may want to head into the Storybook discord for more help! I'm not certain your issue relates to this macro or not, regardless I don't have the bandwidth to resolve your issue!

**Comment by vinipachecov** [comment]
@kylemh  Thanks man, I'll try some help over there.

**Comment by zerubeus** [comment]
Have the same problem any have you found anything @vinipachecov ?

**Comment by vinipachecov** [comment]
@zerubeus Not really. Storybook needs a platform so this require context will not work, it only works for web from what I've understood.

**Comment by MarkJordan66** [comment]
> Hi there!
> 
> 
> 
> I was following the storybook tutorial but using it with react native. Is it working for RN?
> 
> 1. I installed require-context.macro
> 
> 2. I installed babel-plugin-macros
> 
> 3. Configured storybook to load stories
> 
> 
> 
> Here is my config file for storybook
> 
>  
> 
> ![image](https://user-images.githubusercontent.com/2838834/67759775-d7eb5d80-fa1e-11e9-9c2f-25815dadc635.png)
> 
> 
> 
> ![image](https://user-images.githubusercontent.com/2838834/67759826-ee91b480-fa1e-11e9-963a-8c14fc254f7a.png)
> 
> 
> 
> My package.json
> 
> 
> 
> ![image](https://user-images.githubusercontent.com/2838834/67759889-0f5a0a00-fa1f-11e9-909c-79d3409f69d7.png)
> 
> 
> 
> ![image](https://user-images.githubusercontent.com/2838834/67760172-98714100-fa1f-11e9-8ff1-ba59c059546f.png)
> 
> 
> 
> Steps to reproduce:
> 
> - Create react native app with react-native init
> 
> - Install and configure storybook
> 
> - install require-context.macro
> 
> - configure stories as in storybook tutorial
> 
> - see error

**Comment by pke** [comment]
I am not quite understanding how this macro overrides the require function? I can't find such code, so since this is a babel compile time only macro, it should work inside the RN app too.
